Lib/test/test_venv.py

```diff
@@ -150,14 +150,17 @@ def test_prompt(self):
     def test_upgrade_dependencies(self):
         builder = venv.EnvBuilder()
         bin_path = 'Scripts' if sys.platform == 'win32' else 'bin'
-        python_exe = 'python.exe' if sys.platform == 'win32' else 'python'
+        python_exe = os.path.split(sys.executable)[1]
```

Comment:
It can be. `python_d.exe` is the only one we support upstream, but I'd like it to be viable with other names (I have some private builds with alternate executable names).

But right now it doesn't work anyway. The venv launcher executable has hardcoded names in it, so it'll only work as `python.exe`, `python_d.exe`, `pythonw.exe` and `pythonw_d.exe`. Which means that this test is basically correct, unless we one day start running our test suite with a different executable name.